src/filefinder.cpp

```diff
@@ -739,6 +741,132 @@ bool FileFinder::IsEasyRpgProject(DirectoryTree const& dir){
 	return(ldb_it != dir.files.end() && lmt_it != dir.files.end());
 }
 
+bool FileFinder::IsRPG2kProjectWithRenames(DirectoryTree const& dir) {
+	return !GetRPG2kProjectWithRenames(dir).Empty();
+}
+
+FileFinder::RPG2KNonStandardFilenameGuesser FileFinder::GetRPG2kProjectWithRenames(DirectoryTree const& dir) {
+	// Try to rescue and determine file extensions.
+	std::regex rgx("^rpg_rt[.](...)$");
+
+	// We need to figure out the names of the map tree and the DB (maps come later).
+	std::vector<RPG2KNonStandardFilenameGuesser::ExtAndSize> candidates;
+	for (const std::pair<std::string, std::string>& item : dir.files) {
+		// Check for any RPG_RT files
+		std::smatch m;
+		if(std::regex_search(item.first, m, rgx)) {
+			std::string ext = m[1].str();
+			if (ext != "exe" && ext != "ini") {
+				candidates.push_back({item.second, ext, GetFileSize(item.second)});
```

Comment:
this doesn't work when launched from the Gamebrowser (wrong relative path).

Use:
```
candidates.push_back({item.second, ext, GetFileSize(FileFinder::FindDefault(dir, item.second))});
```

Reply:
Fixed in 9e361a9bf1052, thanks